.travis.yml

```diff
@@ -13,12 +10,15 @@ python:
 - 3.6
 - &mainstream_python 3.7
 - nightly
-- &pypy3 pypy3.5-6.0.0
+- pypy3.5-6.0.0
```

Comment:
Also needs `pypy3.6-7.1.1`.

Reply:
I meant to add this to the initial post: I deliberately didn't touch the pypi version either. This can be done in a follow-up PR.